docs/examples/collect.py

```diff
@@ -8,7 +8,10 @@
 import tempfile, shutil, urllib, io, sys, subprocess
 import unittest
 
-tile_url = 'https://tile.mapzen.com/mapzen/terrain/v1/geotiff/{z}/{x}/{y}.tif'
+tile_url = 'https://tile.mapzen.com/mapzen/terrain/v1/geotiff/{z}/{x}/{y}.tif&api_key={k}'
```

Comment:
Aha! That's what I had originally then when I checked https://mapzen.com/documentation/overview/#developer-accounts-and-api-keys it told me to use `&` so I switched. I'll file an issue in the docs to use `?` instead and update here.